nosnoc/model.py

```diff
@@ -353,8 +353,7 @@ def preprocess_model(self, opts: NosnocOpts):
             E= ca.SX.eye(self.dims.n_c_sys)
             f_x =  self.f_unconstrained[0] +  E @ J_c_pds.T @ lam[0]
             g_switching = ca.vertcat(g_switching, ca.vertcat(*self.c_pds) - lam[0])
-            g_z_all = ca.SX([])
-            std_compl_res += ca.transpose(lam[0]) @ ca.vertcat(*self.c_pds)
+            std_compl_res += ca.fabs(ca.transpose(lam[0]) @ ca.vertcat(*self.c_pds))
```

Comment:
You shouldn't need to take the absolute value here, as a matter of fact, doing so can hide bugs.